app/assets/stylesheets/homepage.css.scss

```diff
@@ -196,25 +193,25 @@ list-style-type: none;
 
 #alex.demo-card-square > .mdl-card__title {
     color: #fff;
-    background: url('alex-headshot.jpg') bottom right 15% no-repeat #000000;
+    background: image-url('alex-headshot.jpg') bottom right 15% no-repeat #000000;
     background-size: 200px 200px;
 }
 
 #max.demo-card-square > .mdl-card__title {
     color: #fff;
-    background: url('max-headshot.jpg') top right 15% no-repeat #000000;
+    background: image-url('max-headshot.jpg') top right 15% no-repeat #000000;
     background-size: 200px 200px;
 }
 
 #david.demo-card-square > .mdl-card__title {
     color: #fff;
-    background: url('david-headshot.jpg') bottom right 15% no-repeat #000000;
+    background: image-url('david-headshot.jpg') bottom right 15% no-repeat #000000;
     background-size: 200px 200px;
 }
 
 #adam.demo-card-square > .mdl-card__title {
     color: #fff;
-    background: url('adam-headshot.jpg') bottom right 15% no-repeat #000000;
+    background: image-url('adam-headshot.jpg') bottom right 15% no-repeat #000000;
```

Comment:
Color literals like `#000000` should only be used in variable declarations; they should be referred to via variable everywhere else.<br>Line should be indented 2 spaces, but was indented 4 spaces<br>Prefer double-quoted strings